src/main/cpp/jni/javabridge_api.h

```diff
@@ -20,38 +20,23 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-#ifndef JAVABRIDGE_API_H_INCLUDED
-#define JAVABRIDGE_API_H_INCLUDED
+#ifndef __MWENGINE__JAVABRIDGE_API_H_INCLUDED
+#define __MWENGINE__JAVABRIDGE_API_H_INCLUDED
 
 #include <jni/javabridge.h>
```

Comment:
JavaBridge API is not wrapped inside a namespace as it will expose (a single) method inside the built SWIG library. This file acts as a magic glue between JNI and native layers.